src/wp-admin/contribute.php

```diff
@@ -0,0 +1,132 @@
+<?php
+/**
+ * Contribute administration panel.
+ *
+ * @package WordPress
+ * @subpackage Administration
+ */
+
+/** WordPress Administration Bootstrap */
+require_once __DIR__ . '/admin.php';
+
+// Used in the HTML title tag.
+$title = __( 'Get Involved' );
+
+list( $display_version ) = explode( '-', get_bloginfo( 'version' ) );
+
+require_once ABSPATH . 'wp-admin/admin-header.php';
+?>
+<div class="wrap about__container">
+
+	<div class="about__header">
+		<div class="about__header-title">
+			<h1>
+				<?php _e( 'Get Involved' ); ?>
+			</h1>
+		</div>
+
+		<div class="about__header-text">
+			<?php _e( 'Be the future of WordPress' ); ?>
+		</div>
+	</div>
+
+	<nav class="about__header-navigation nav-tab-wrapper wp-clearfix" aria-label="<?php esc_attr_e( 'Secondary menu' ); ?>">
+		<a href="about.php" class="nav-tab"><?php _e( 'What&#8217;s New' ); ?></a>
+		<a href="credits.php" class="nav-tab"><?php _e( 'Credits' ); ?></a>
+		<a href="freedoms.php" class="nav-tab"><?php _e( 'Freedoms' ); ?></a>
+		<a href="privacy.php" class="nav-tab"><?php _e( 'Privacy' ); ?></a>
+		<a href="contribute.php" class="nav-tab nav-tab-active" aria-current="page"><?php _e( 'Get Involved' ); ?></a>
+	</nav>
+
+	<div class="about__section has-2-columns is-wider-right">
+		<div class="column about__image">
+			<img src="data:image/svg+xml,%3Csvg width='436' height='436' viewbox='0 0 100 100' xmlns='http://www.w3.org/2000/svg'%3E%3Crect width='100%25' height='100%25' fill='%231d35b4' /%3E%3C/svg%3E" alt="" />
+		</div>
+		<div class="column is-vertically-aligned-center">
+			<p>
+				<?php
+				printf(
+					/* translators: %s: Percentage value. Note: "%%" is used to display an escaped percentage sign. */
+					__( 'Do you use WordPress for work, for personal projects, or maybe even just for fun? You can help shape the long-term success of the open source project that powers %s%% of the web.' ),
+					43
```

Comment:
I am wondering if the exact percentage should be left out.  This number will change constantly, and this is probably better summarized as "millions of websites".

Reply:
This was discussed in the draft document and for now is a confirmed decision. Possibly @sereedmedia can explain why

Reply:
> I am wondering if the exact percentage should be left out.

Agreed. Since Matt and Josepha shared that this metric is increasingly inaccurate, we have been actively moving away from using it in copy across WordPress.org. I highly recommend removing it.

Reply:
@eidolonnight, @DanSoschin can you please communicate with the rest of the team on the draft document? I was asking the same question, but this percentage was kept. I have no idea why. Please…

Reply:
Done. Thank you @OlaIola !

Reply:
this has been updated!

Reply:
the % was added by me (found it on a page on make only), as I felt the intro was not strong. 
I did not know that we were working away from using it in copy. Apologies @OlaIola. 
I will keep that in mind.